Assets/_Scripts/Managers/NightManager.cs

```diff
@@ -157,6 +163,13 @@ void StartNight(PerformanceDataSO performanceDataSo)
     public void EndNight()
```

Comment:
This function has no references, is it referenced by the Unity editor? is it zombie code? a stub?